### PR TITLE
Lazyloading fix for pageskin

### PR DESCRIPTION
**Pull request by Calanthe**
This PR leaves lazy loading on for non-wide breakpoints when a pageskin campaign is serving.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/dfp.js [hunk 1]
```diff
@@ -292,7 +292,7 @@ define([
             window.googletag.cmd.push(defineSlots);
 
             // We do not want lazy loading on pageskins because it messes up the roadblock
-            if (config.switches.viewability && !config.page.hasPageSkin) {
+            if (config.switches.viewability && (!config.page.hasPageSkin || (config.page.hasPageSkin && detect.getBreakpoint() !== 'wide'))) {
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/common/modules/commercial/dfp.js]: Isn't this the same as `... && (!(config.page.hasPageSkin && detect.getBreakpoint() === 'wide')))` ?

Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/dfp.js]: Yes, you are right. I will make changes.

Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/commercial/dfp.js]: I think this logic is getting pretty convoluted in places

File: static/src/javascripts/projects/common/modules/commercial/dfp.js [hunk 2]
```diff
@@ -680,10 +683,11 @@ define([
...
         dfp = {
-            init:        init,
-            addSlot:     addSlot,
-            refreshSlot: refreshSlot,
-            getSlots:    getSlots,
+            init:           init,
+            addSlot:        addSlot,
+            refreshSlot:    refreshSlot,
+            getSlots:       getSlots,
+            shouldLazyLoad: shouldLazyLoad,
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/modules/commercial/dfp.js]: Given that this is a private method, I think we should either name it as such with a dangling underscore or extract it to a different file, like dfp-helpers.js. But it should not look like an ordinary public interface.

**Comment by jbreckmckye** [comment]
:+1:

**Pull request merged**